### DISCUSSION TITLE
Trouble in GE Signa PET/MR Example Recon

### DISCUSSION THREAD

**Discussion by dimitri-mantovani** [UCL/STIR#1296] [Q&A]
Hi! I am using the VM of the SIRF software (3.5.0) adapted to include the STIR software (5.1.0) (**update_VM_to_full_STIR.sh**), to learn how to reconstruct the GE Signa raw data. However, I am having some problems when trying to process the example data provided in the "_buildVM/sources/STIR/examples/GE-Signa-PETMR_" folder.

My problem is occurring in the reconstruction step without attenuation correction (**NAC_recon.sh**). The following result appears in my terminal:

***
_[......]
INFO: Thread 1/5 calculating segment_num: 19, view_num: 96
INFO: Thread 2/5 calculating segment_num: 19, view_num: 110
INFO: Thread 4/5 calculating segment_num: 19, view_num: 124
INFO: Thread 3/5 calculating segmeKilled
WARNING: image duration not set, so time frame definitions will not be initialised
 the axis to invert is x
ERROR: Error opening file mrac.hv
terminate called after throwing an instance of 'std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >'
Aborted (core dumped)_
***
I'm quite a beginner in C++, so I'm finding it difficult to understand what this error might be. Thanks! :)

**Comment by KrisThielemans** [comment]
Hi. Sorry for missing this post.

I cannot reproduce this. There is no reference to a `mrac.hv` in the current STIR or v5.1.0 except *after* the `exit` in https://github.com/UCL/STIR/blob/75a9342157e59db6d43a968fa48f7d8c951853fb/examples/GE-Signa-PETMR/process_VQC_data.sh#L62
so it never gets to the registration step. This is because we didn't get an mrac for this data.

If this doesn't solve your problem, could you give some more detail on what you're trying to do and how?
